ProjectOne/sass/pages/_home.scss

```diff
@@ -141,7 +141,158 @@
 }
 
 .featured-products {
-  background-color: brown;
+  background-image: url("../assets/images/featured-production/10.png");
+
+  .title {
+    &__categories {
+      text-align: center;
+    }
+
+    &__category {
+      @include styleList($color: #c4c4c4, $font-size: 1.3rem);
+      padding: 1rem 1rem 1rem 1.5rem;
+    }
+
+    &__category:nth-of-type(2) {
```

Comment:
Check lại biến màu + sử dụng class khác để style cho chứ tránh sử dụng nth-of-type nhé
Vì nếu có thay đổi số lượng item thì nth-of-type lúc này nó k còn đúng nữa